### PR TITLE
publications on two way alternating automata

### PR DESCRIPTION
**Pull request by susuhahnml**

### CODE REVIEW

File: krr.bib [hunk 1]
```diff
@@ -23097,6 +23147,15 @@ @inproceedings{vardi97a
   pages = {191-206}
 }
 
+@inproceedings{vardi98a,
+  title = {Reasoning about the past with two-way automata},
+  author = {M. Vardi},
+  booktitle = {International Colloquium on Automata, Languages, and Programming},
+  organization = {Springer},
```
Comment (tortinator) [inline krr.bib:23154]:
```suggestion
  organization = springer,
```

**Review by tortinator — APPROVED** [review]
Thanks, I corrected a beauty spot right away

File: krr.bib (hunk 1 above)
Comment (rkaminsk) [reply krr.bib:23154]: @tortinator, the PR had page ranges using `--` instead of `-`, which the remainder of the bibliography uses.

Comment (tortinator) [reply krr.bib:23154]: Ok, I'll fix it

Comment (rkaminsk) [reply krr.bib:23154]: I could also easily replace the double dash in the bibfmt script.

Comment (tortinator) [reply krr.bib:23154]: > I could also easily replace the double dash in the bibfmt script.

Would be good

**Pull request merged**